back-end/routes/auth.js

```diff
@@ -0,0 +1,123 @@
+const express = require('express')
+const app = require('express').Router()
+const User=require('../models/User')
+const passport = require('passport');
+const localStrategy = require('passport-local').Strategy;
+
+const bcrypt = require('bcrypt') //encrypt password
+const { body, validationResult } = require('express-validator')
+const expressSession = require('express-session') 
+app.use(express.urlencoded({ extended: true }))
+app.use(expressSession({'secret':'any','saveUninitialized':false, 'resave':false}))
```

Comment:
This is more a question than a change request, but does the code work if `expressSession` is removed? If so, then could you remove references to it?